digits/model/images/generic/views.py

```diff
@@ -153,11 +153,8 @@ def create(extension_id=None):
                         elif epoch == -1:
                             pretrained_model = old_job.train_task().pretrained_model
                         else:
-                            for filename, e in old_job.train_task().snapshots:
-                                if e == epoch:
-                                    pretrained_model = filename
-                                    break
-
+                            # verify snapshot exists
+                            pretrained_model = old_job.train_task().get_snapshot(epoch, download=True)
```

Comment:
This `download` flag comes from the fact that Tensorflow appends the `.data-00000-of-00001` suffix to the files it creates. So if you do something like `tf.save('toto')` it is going to create a file named `toto.data-00000-of-00001`. If you want to *download* the snapshot you need to get the file `toto.data-00000-of-00001`. The complication comes from the fact that if you want to *reload* the snapshot in Tensorflow you can't do `tf.load('toto.data-00000-of-00001')` you need to do `tf.load('toto')`... 

I could have implemented this change differently and referred to the actual filename in `get_snapshot()` and I would just have had to script the `data-00000-of-00001` when reloading the snapshot in TF. That would have isolated the change to the TF implementation. I guess I didn't think it through thoroughly at the time. Didn't really expect to ever release this though :-)